src/benchmarklib/sqlite_add_indices.cpp

```diff
@@ -0,0 +1,54 @@
+#include "sqlite_add_indices.hpp"
+
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "benchmark_runner.hpp"
+#include "storage/storage_manager.hpp"
+#include "utils/sqlite_wrapper.hpp"
+#include "utils/timer.hpp"
+
+namespace opossum {
+
+void add_indices_to_sqlite(const std::string& schema_file_path, const std::string& create_indices_file_path,
+                           BenchmarkRunner& benchmark_runner) {
```

Comment:
should take "sqlite_wrapper" as a param, uses nothing else of the benchmark runner. is this tested?

Reply:
Now it is.